app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
Okay so maybe we shouldn't be inventorying catalog_items as images, instead inventory `vapp_templates` as `OrchestrationTemplates` and then the VMs under a `vapp_template` can be associated with the `OrchestrationTemplate`